### ISSUE TITLE
Preprocessing multi labeled data

### ISSUE THREAD

**Issue by efferre99** [keras-team/keras-applications#44]
I have to preprocess images that have multi label.
In particular I have to project a neural network that not only classifies two different shape, triangle or circle, but also get their dimensions: the side, the radius, the position etc.. It's seems with no sense, but for my application it has. Where can I find a similar example?

**Comment by taehoonlee** [comment]
@efferre99, How about [DeepPose (CVPR 2014)](https://www.cv-foundation.org/openaccess/content_cvpr_2014/papers/Toshev_DeepPose_Human_Pose_2014_CVPR_paper.pdf)? There are various approaches to estimate attributes such as a radius and a position.

**Comment by taehoonlee** [comment]
@efferre99, I'll close the issue for now. Please feel free to open it again at any time if you have additional comments.
